aspnetcore/blazor/webassembly-lazy-load-assemblies.md

```diff
@@ -339,10 +339,6 @@ Add an ASP.NET Core class library project to the solution:
 * Visual Studio: Right-click the solution file in **Solution Explorer** and select **Add** > **New project**. From the dialog of new project types, select **Razor Class Library**. Name the project `GrantImaharaRobotControls`. Do **not** select the **Support pages and view** checkbox.
 * Visual Studio Code/.NET CLI: Execute `dotnet new razorclasslib -o GrantImaharaRobotControls` from a command prompt. The `-o|--output` option creates a folder and names the project `GrantImaharaRobotControls`.
 
-The example component presented later in this section uses a [Blazor form](xref:blazor/forms/index). In the RCL project, add the [`Microsoft.AspNetCore.Components.Forms`](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.Forms) package to the project.
```

Comment:
It might be better to state what's going on with the packages here. This could instead say something like ...

> The example component presented later in this section uses a \[Blazor form](xref:blazor/forms/index). Because the RCL was created without supporting pages and views, the RCL includes a package reference for \[\`Microsoft.AspNetCore.Components.Web\`](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.Web), which transitivity includes the \[\`Microsoft.AspNetCore.Components.Forms\`](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.Forms) package.
>
> \[!INCLUDE\[](~/includes/package-reference.md)]

Reply:
If you create the RCL with "Support pages and view" checked, the new project still successfully compiles the `Robot.razor` component (with `<EditForm>` and `<InputRadioGroup>`), most likely because of `<FrameworkReference Include="Microsoft.AspNetCore.App" />`.

Why do you think we need to include package-related info in the article?

Reply:
Truly, it was just a passing concern. I didn't know what would happen in that configuration. Let's go ahead with your suggestion.